test/unittest/dds/publisher/CMakeLists.txt

```diff
@@ -298,7 +297,6 @@ set(DATAWRITERTESTS_SOURCE DataWriterTests.cpp
     endif(TINYXML2_INCLUDE_DIR)
 
     set(PUBLISHERTESTS_SOURCE PublisherTests.cpp
-    ${PROJECT_SOURCE_DIR}/src/cpp/dds/pub/DataWriter.cpp
```

Comment:
No, this the `src/cpp/dds` folder included source files implementing the PSM API which is not officially supported by Fast DDS yet. It might be recuperated and fully implemented in the future.